lib/matplotlib/tests/baseline_images/test_image/image_shift.svg

```diff
@@ -26,9 +26,9 @@ L 72 43.2
+   <g clip-path="url(#pa943344acd)">
+    <image height="99" id="image784d7f5ce5" transform="matrix(4.525253 0 0 3.505051 72 41.8)" width="99" xlink:href="data:image/png;base64,
```

Comment:
Yes -- the new representation is to store the exact image size in `width` and `height` and then scale in the transform matrix.  This basically fell out from the simplification of the `draw_image` API to just pass a transform rather than scale, offset and transform (which was redundant).  It all amounts to the same thing at the end of the day.